scripts/issue/12/seed_rembadger.py

```diff
@@ -0,0 +1,43 @@
+from brownie import interface
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+
+TO_SEED = 600_000
+
+
+def main():
+    """
+    seed rembadger vault with 600k $badger
+    """
+
+    safe = GreatApeSafe(registry.eth.badger_wallets.treasury_vault_multisig)
+    rembadger = interface.IRemBadger(registry.eth.sett_vaults.remBADGER)
+    verifier = interface.IBalanceVerifier(
+        registry.eth.helpers.balance_checker, owner=safe.account
+    )
+    badger = interface.ERC20(
+        registry.eth.treasury_tokens.BADGER, owner=safe.account
+    )
+
+    safe.take_snapshot(tokens=[badger.address])
+
+    mantissa = TO_SEED * 10 ** badger.decimals()
+    assert badger.balanceOf(safe) >= mantissa
+
+    # get current balance of rembadger vault
+    bal_before = badger.balanceOf(rembadger)
+    pps_before = rembadger.getPricePerFullShare()
+
+    # transfer
+    badger.approve(rembadger, mantissa)
```

Comment:
i think here the approve is not needed

Reply:
Agreed

Reply:
ah yea thats still a relic from when it wasnt `transfer` but `addWant`.